lib/compat/wordpress-6.9/class-gutenberg-rest-old-templates-controller.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+class Gutenberg_REST_Old_Templates_Controller extends WP_REST_Templates_Controller {
```

Comment:
Hi @ellatrix I need to extend WP_REST_Templates_Controller for another feature. 

Can I piggy back off `Gutenberg_REST_Old_Templates_Controller`? 

Could you explain how it relates to `Gutenberg_REST_Templates_Controller`? Some comments in the code might help folks working on 6.9.

Reply:
As we discussed, it would make sense to add the method to override for all 3 controllers if you need to filter templates.